### PR TITLE
DB-8627 eNSDS (3.0)

### PR DESCRIPTION
**Pull request by jpanko1**

### CODE REVIEW

File: splice_machine/pom.xml [hunk 1]
```diff
@@ -165,6 +165,22 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka_${scala.binary.version}</artifactId>
+<!--            <version>${kafka.version}</version>-->
+            <version>0.10.0-kafka-2.1.2</version>
```
Comment (OlegMazurov) [inline splice_machine/pom.xml:172]: splice_machine is a core module, i.e. platform independent.
Kafka dependency is platform dependent. If we build against 0.10.0-kafka-2.1.2 but the run-time dependency is different and not fully compatible, we may run into obscure errors.

Comment (jpanko1) [reply splice_machine/pom.xml]: Removed Kafka dependencies from splice_machine in commit [67b33b0](https://github.com/splicemachine/spliceengine/pull/3405/commits/67b33b00e537df5b9dfe5c4acc4ff7013ce1affa)

File: splice_spark2/src/main/spark2.2/com/splicemachine/spark2/splicemachine/KafkaUtils.scala [hunk 2]
```diff
@@ -0,0 +1,58 @@
+package com.splicemachine.spark2.splicemachine
```
Comment (carolp-503) [inline splice_spark2/src/main/spark2.2/com/splicemachine/spark2/splicemachine/KafkaUtils.scala:1]: Copyright?  Not a blocker for bringing this in, but should be added at some point.

File: splice_spark2/src/main/spark2.2/com/splicemachine/spark2/splicemachine/SpliceJDBCUtil.scala [hunk 3]
```diff
@@ -0,0 +1,212 @@
+package com.splicemachine.spark2.splicemachine
```
Comment (carolp-503) [inline splice_spark2/src/main/spark2.2/com/splicemachine/spark2/splicemachine/SpliceJDBCUtil.scala:1]: Copyright?  Not a blocker for bringing this in, but should be added at some point.

File: splice_spark2/src/main/spark2.2/org/apache/spark/sql/execution/datasources/jdbc/SpliceRelation2.scala [hunk 4]
```diff
@@ -0,0 +1,68 @@
+package org.apache.spark.sql.execution.datasources.jdbc
```
Comment (carolp-503) [inline splice_spark2/src/main/spark2.2/org/apache/spark/sql/execution/datasources/jdbc/SpliceRelation2.scala:1]: Copyright?  Not a blocker for bringing this in, but should be added at some point.

File: splice_spark2/src/main/spark2.2/org/apache/spark/sql/execution/datasources/jdbc/SplicemachineDialect2.scala [hunk 5]
```diff
@@ -0,0 +1,41 @@
+package org.apache.spark.sql.execution.datasources.jdbc
```
Comment (carolp-503) [inline splice_spark2/src/main/spark2.2/org/apache/spark/sql/execution/datasources/jdbc/SplicemachineDialect2.scala:1]: Copyright?  Not a blocker for bringing this in, but should be added at some point.

File: splice_spark2/src/main/spark2.3/com/splicemachine/spark2/splicemachine/KafkaUtils.scala [hunk 6]
```diff
@@ -0,0 +1,58 @@
+package com.splicemachine.spark2.splicemachine
```
Comment (carolp-503) [inline splice_spark2/src/main/spark2.3/com/splicemachine/spark2/splicemachine/KafkaUtils.scala:1]: Copyright?  Not a blocker for bringing this in, but should be added at some point.

File: splice_spark2/src/main/spark2.3/com/splicemachine/spark2/splicemachine/SpliceJDBCUtil.scala [hunk 7]
```diff
@@ -0,0 +1,212 @@
+package com.splicemachine.spark2.splicemachine
```
Comment (carolp-503) [inline splice_spark2/src/main/spark2.3/com/splicemachine/spark2/splicemachine/SpliceJDBCUtil.scala:1]: Copyright?  Not a blocker for bringing this in, but should be added at some point.

File: splice_spark2/src/main/spark2.3/org/apache/spark/sql/execution/datasources/jdbc/SpliceRelation2.scala [hunk 8]
```diff
@@ -0,0 +1,68 @@
+package org.apache.spark.sql.execution.datasources.jdbc
```
Comment (carolp-503) [inline splice_spark2/src/main/spark2.3/org/apache/spark/sql/execution/datasources/jdbc/SpliceRelation2.scala:1]: Copyright?  Not a blocker for bringing this in, but should be added at some point.

File: splice_spark2/src/main/spark2.3/org/apache/spark/sql/execution/datasources/jdbc/SplicemachineDialect2.scala [hunk 9]
```diff
@@ -0,0 +1,41 @@
+package org.apache.spark.sql.execution.datasources.jdbc
```
Comment (carolp-503) [inline splice_spark2/src/main/spark2.3/org/apache/spark/sql/execution/datasources/jdbc/SplicemachineDialect2.scala:1]: Copyright?  Not a blocker for bringing this in, but should be added at some point.

**Pull request merged**